packages/components/src/components/progress/component.tsx

```diff
@@ -181,12 +192,28 @@ export class KolProcess implements Generic.Element.ComponentApi<RequiredProps, O
 		});
 	}
 
+	@Watch('_variant')
+	public validateVariant(value?: KoliBriProgressType): void {
+		if (!value && this._type) {
+			// remove with v2
+			value = this._type;
+		}
+		if (value !== 'cycle') {
+			value = 'bar';
+		}
+		this.state = {
+			...this.state,
+			_variant: value,
+		};
+	}
+
 	public componentWillLoad(): void {
 		this.validateLabel(this._label);
 		this.validateMax(this._max);
 		this.validateType(this._type);
```

Comment:
Der Aufruf kann raus.

Reply:
Du meinst `this.validateType(this._type);`? Dann würde _type doch nicht mehr funktionieren?